lib/order.rb

```diff
@@ -0,0 +1,29 @@
+require_relative './menu.rb'
+
+class Order
```

Comment:
You can initialize the menu in order class so you don't have to have it twice.

def initialise(menu = Menu.new)
@menu = menu
end